ui/hooks/spending.ts

```diff
@@ -106,6 +107,9 @@ export function useUpdateSpending(): (_spending: Spending) => Promise<void> {
             previous.map(item => item.spendingId === updatedSpending.spendingId ? updatedSpending : item),
         ),
         queryClient.invalidateQueries(`/bank_accounts/${ updatedSpending.bankAccountId }/balances`),
+        queryClient.invalidateQueries(
+          `/bank_accounts/${ updatedSpending.bankAccountId }/spending/${ updatedSpending.spendingId }/funding`,
+        ),
         queryClient.invalidateQueries([`/bank_accounts/${ updatedSpending.bankAccountId }/forecast/next_funding`]),
```

Comment:
The query key for this one is an array, but can we just use the string? If we can we can make these 3 invalidates a single one.